### PR TITLE
Adding inotify-tools to universal image

### PR DESCRIPTION
**Pull request by osortega**
This PR adds `inotify-tools` to the list of packages to be installed for the universal image.

### CODE REVIEW

**Review by joshspicer — APPROVED** [review]
LGTM

File: src/universal/.devcontainer/Dockerfile [hunk 1]
```diff
@@ -55,6 +55,7 @@ RUN apt-get update \
         uuid-dev \
         curl \
         gettext \
+        inotify-tools \
```
Comment (samruddhikhandale) [inline src/universal/.devcontainer/Dockerfile:58]: Can you add a test?
 
You can add this package to the list [here](https://github.com/devcontainers/images/blob/c4220a0e94143536bcbe9490662e8e47ba2fb43f/src/universal/test-project/test-utils.sh#L133) for testing. (This way we can validate if things change in future (like soon we might change the base image from focal to bookworm, would be good to have a validation in place.)

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Left one minor comment, otherwise it looks great!

**Review by samruddhikhandale — APPROVED** [review]
Thanks! I'll merge this once tests are 🟢

**Pull request merged**